src/execution_plan/execution_plan.c

```diff
@@ -522,6 +523,18 @@ static void _ExecutionPlanSegment_PlaceFilterOps(ExecutionPlanSegment *segment)
 	Vector_Free(sub_trees);
 }
 
+// Check whether an array of operations contains a Create op.
+static inline bool _ExecutionPlan_ContainsCreateOp(OpBase *root) {
```

Comment:
I think this function can be more generic: `_ExecutionPlan_ContainsOp(OpBase *root, OPType type)`